the-collab-lab/tcl-38-smart-shopping-list
# Refactor firebase snapshot, place only in app.js, update test, edit h…

## Description

In this PR, we refactored the following:

- useFirebaseSnapshot hook has a token dependency, only brings back docs for a specific token
- usage of hook is limited to app.js and passes items as props to itemlist and additems
- adjusted the tokenContext to surround the app component
- updated the itemList test to accept items as a prop
- To address a console error from prior styling work, we changed a `p` tag to a `div` in itemList.

(please note, firebase.js is currently set to our temp.db due to overage on original db)

## Related Issue

closes #70 

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
| ✓   | :bug: Bug fix              |
|   | :sparkles: New feature     |
| ✓  | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Testing Steps / QA Criteria

Pull down branch. App should function as normal. You should not see any changes in behavior.
